README.md

```diff
@@ -11,3 +12,19 @@ This project aims to deal with modules related to the webclient of Odoo. You'll
 - Add widgets
 - Ease the import/export features
 - Generally add clientside functionality
+
+[//]: # (addons)
+This part will be replaced when running the oca-gen-addons-table script from OCA/maintainer-tools.
+[//]: # (end addons)
```

Comment:
For now, it's intended only for >8.0 branch, but as a placeholder, there's no problem to have it here in case the script is also run here.